pages/components/Modal/modal.css

```diff
@@ -0,0 +1,43 @@
+@import url(/css/main.css);
+
+.modal-preview-container {
+    display: flex;
+    flex-direction: column;
+    height: fit-content;
+    border: 1px solid var(--secondary-faded-color);
+    border-radius: 8px;
+    padding: 1rem;
+    justify-content: space-evenly;
+    align-items: center;
+    gap: 1rem;
+}
+
+.modal{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
```

Comment:
The default value of flex-wrap is nowrap. I guess you can omit this line.

Reply:
Ok thanks will add that !